x-pack/legacy/plugins/actions/server/init.ts

```diff
@@ -24,17 +24,36 @@ import { createOptionalPlugin } from '../../../server/lib/optional_plugin';
 
 export function init(server: Legacy.Server) {
   const config = server.config();
+  const taskManager = server.plugins.task_manager!;
+  const { callWithRequest } = server.plugins.elasticsearch.getCluster('admin');
   const spaces = createOptionalPlugin<SpacesPlugin>(
     config,
     'xpack.spaces',
     server.plugins,
     'spaces'
   );
 
-  const { callWithInternalUser } = server.plugins.elasticsearch.getCluster('admin');
-  const savedObjectsRepositoryWithInternalUser = server.savedObjects.getSavedObjectsRepository(
-    callWithInternalUser
-  );
+  server.plugins.xpack_main.registerFeature({
+    id: 'actions',
+    name: 'Actions',
+    app: ['actions', 'kibana'],
+    privileges: {
+      all: {
```

Comment:
We should restrict access to the Actions APIs to only users who have this feature: https://github.com/elastic/kibana/blob/2b4f2a259e02f2a788d374a88b036c0444c44ed3/x-pack/legacy/plugins/xpack_main/server/lib/feature_registry/feature_registry.ts#L33-L57